trace/opencensus/trace/ext/flask/flask_middleware.py

```diff
@@ -49,7 +49,8 @@ class FlaskMiddleware(object):
                      :class:`.PrintExporter`. The rest option is
                      :class:`.FileExporter`.
     """
-    def __init__(self, app, sampler=None, exporter=None, propagator=None):
+    def __init__(self, app, sampler=None, exporter=None, propagator=None,
+                 transport=None):
```

Comment:
I think you didn't mean to have `transport` in this PR?

Reply:
Removed.